providers/amazon/src/airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -565,7 +565,13 @@ def _log_results(self, result: dict[str, Any]) -> None:
         Rule_3    ColumnLength "marketplace" between 1 and 2     FAIL        {'Column.marketplace.MaximumLength': 9.0, 'Column.marketplace.MinimumLength': 3.0}     Value: 9.0 does not meet the constraint requirement!
 
         """
-        import pandas as pd
+        try:
+            import pandas as pd
```

Comment:
Should we add pandas as an extra of amazon provider?

Reply:
My only concern would be that pandas is a very very large dependency and it's only a very small/rare piece of our code that uses it. If we added it as an extra then it would start being installed automatically for anyone doing an `--all-extras` install of the amazon provider package, which may negatively impact those users. However, that's a risk they're taking if they're doing such an install? Curious what others think.